### PR TITLE
Add e2e tests for  organization.

### PR DESCRIPTION
**Pull request by JoyLubega**
Resolve: opencollective/opencollective/issues/2451
This PR add e2e tests for Organization

### CODE REVIEW

**Comment by JoyLubega** [comment]
@znarf I am done rebasing

**Comment by znarf** [comment]
This looks good. Unfortunately, @Betree is the reviewer for that and he will not be able to look at it until November 12th, so we'll have to wait to merge.

File: test/cypress/integration/04-organization_v2.test.js [hunk 1]
```diff
@@ -3,17 +3,34 @@ describe('New organization profile', () => {
...
 
+      cy.visit(`/${collectiveSlug}/`);
+    });
+  });
   describe('Contributors section', () => {
-    it.skip('Only shows core contributors without any filter', () => {
-      // TODO
+    it('Only shows core contributors without any filter', () => {
+      cy.getByDataCy('section-contributors').click({ force: true });
+      cy.hash().should('eq', '#section-contributors');
```
Comment (Betree) [inline test/cypress/integration/04-organization_v2.test.js]: Great that you checked that!

Comment (JoyLubega) [reply test/cypress/integration/04-organization_v2.test.js]: Hey @Betree  I need more clarity on the changes required

**Comment by JoyLubega** [comment]
@Betree  I have made the changes

File: test/cypress/integration/04-organization_v2.test.js (hunk 1 above)
Comment (Betree) [reply test/cypress/integration/04-organization_v2.test.js]: No changes required, I was just saying that this was a great change :-)

File: test/cypress/integration/04-organization_v2.test.js [hunk 2]
```diff
@@ -3,17 +3,35 @@ describe('New organization profile', () => {
...
   describe('Contributors section', () => {
-    it.skip('Only shows core contributors without any filter', () => {
-      // TODO
+    it('Only shows core contributors without any filter', () => {
+      cy.getByDataCy('filters').should('not.exist');
+      cy.getByDataCy('section-contributors').click({ force: true });
+      cy.hash().should('eq', '#section-contributors');
+
+      cy.getByDataCy('section-contributors-title').contains('Core contributors');
+      cy.getByDataCy('ContributorsGrid_ContributorCard');
```
Comment (Betree) [inline test/cypress/integration/04-organization_v2.test.js]: Not necessary (we already check that below)

```suggestion
```

Comment (JoyLubega) [reply test/cypress/integration/04-organization_v2.test.js]: Let me remove it ASAP.

**Comment by JoyLubega** [comment]
@Betree Done!

**Pull request merged**